nengo/synapses.py

```diff
@@ -466,8 +560,332 @@ class Alpha(LinearFilter):
     tau = NumberParam("tau", low=0)
 
     def __init__(self, tau, **kwargs):
-        super().__init__([1], [tau ** 2, 2 * tau, 1], **kwargs)
         self.tau = tau
+        super().__init__(([1], [tau ** 2, 2 * tau, 1]), analog=True, **kwargs)
+
+    @property
+    def cutoff(self):
+        """Cutoff frequency in Hz; frequencies above this are attenuated."""
+        return np.sqrt(np.sqrt(2) - 1) / (2 * np.pi * self.tau)
+
+
+class DoubleExp(LinearFilter):
+    r"""A second-order (two-pole) lowpass filter synapse with no zeros.
+
+    The transfer function is:
+
+    .. math::
+
+        h(t) &= \frac{\exp(-t / \tau_1) - \exp(-t / \tau_2)}{\tau_1 - \tau_2} \\
+        H(s) &= \frac{1}{(\tau_1 s + 1)(\tau_2 s + 1)}
```

Comment:
I added the `h(t)` equation ([see validation](https://www.wolframalpha.com/input/?i=InverseLaplace%5B%5Cfrac%7B1%7D%7B%28%5Ctau_1+s+%2B+1%29%28%5Ctau_2+s+%2B+1%29%7D%5D)) to mirror how they other synapses are presented. Note that even though there is a divide-by-zero when `tau_1 == tau_2` you can take the limit from either side to show that it equals the alpha synapse.